.github/scripts/get-version.cjs

```diff
@@ -0,0 +1,38 @@
+const { execSync } = require('child_process');
+
+async function getVersion(url) {
+  try {
+    const response = execSync(`curl -s '${url}/graphql' -H 'content-type: application/json' --data-raw '{"query":"{ statistics { appVersion } }"}'`);
+    const parsedResponse = JSON.parse(response);
+    const appVersion = parsedResponse.data.statistics.appVersion;
+    return appVersion;
+  } catch (error) {
+    console.error(`Error fetching version from ${url}: ${error.message}`);
+    return null;
+  }
+}
+
+async function main() {
+  const arg = process.argv[2];
+  let urls;
+
+  if (arg === 'mainnet') {
+    const mainnetModule = await import('../../config/mainnet.js');
+    urls = mainnetModule.default.network.rest;
+  } else if (arg === 'fairground') {
+    urls = ['https://api.n07.testnet.vega.rocks', 'https://api.n08.testnet.vega.rocks'];
```

Comment:
Yeah it shouldn't be. This was because test config did not have fairground data nodes declared but I totally forgot we have the testnet config, will change